forms-flow-nav/src/sidenav/Sidebar.scss

```diff
@@ -142,43 +158,60 @@ $animSpeed: 300ms;
       border-bottom-left-radius: 0 !important;
       border-bottom-right-radius: 0 !important;
     }
+    &:last-of-type{
+      border-bottom: none;
+    }
+  }
+
+  .accordion-item:last-of-type{
+    border-radius: 0;
   }
 
   .accordion-link {
     display: block;
     width: 100%;
     
-    padding: $navPadding;
-    padding-left: calc($navPadding + $navSpacing + $base);
+    padding: $navPadding 0;
+    // padding-left: calc($navPadding + $navSpacing + $base);
+    padding-left: 1.44rem;
 
-    font-size: $fontSmallest;
-    font-weight: 700;
+    // font-size: $fontSmallest;
+    // font-weight: 700;
     line-height: $fontLineHeight;
-    color: var(--navbar-main-menu-active-font-color);
+    // color: var(--navbar-main-menu-active-font-color);
     text-align: left;
     text-decoration: none;
     cursor: pointer;
 
+    color: #7C7D7F;
+    font-size: 0.875rem;
+
     &:hover {
       text-decoration: none;
-      background-color: var(--navbar-hover-submenu-bg-color);
-      color: var(--navbar-hover-submenu-font-color);
-      font-size: var(--navbar-hover-submenu-font-size) !important;
-      font-weight: var(--navbar-hover-submenu-font-weight) !important;
+      // background-color: var(--navbar-hover-submenu-bg-color);
+      // color: var(--navbar-hover-submenu-font-color);
+      // font-size: var(--navbar-hover-submenu-font-size) !important;
+      // font-weight: var(--navbar-hover-submenu-font-weight) !important;
+      color: #525254;
+      font-weight: 500;
     }
 
     &.active {
-      background-color: var(--navbar-active-submenu-bg-color);
-      color: var(--navbar-active-submenu-font-color);
-      font-size: var(--navbar-active-submenu-font-size) !important;
-      font-weight: var(--navbar-active-submenu-font-weight) !important;
+      color: #525254;
+      font-size: 0.875rem;
```

Comment:
Please use the mixin here

 @include font-style(--font-size-xl, --font-weight-semibold, --red-100  );